### PR TITLE
ready(obj) directly

### PR DESCRIPTION
**Pull request by fengmk2**
`ready(obj)` should also work the same to `ready.mixin(obj)`, it should work like most modules on node.js.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "url": "git://github.com/SuperShabam/ready.git"
   },
   "engines": {
-    "node": "~0.6.10"
+    "node": ">=0.6.10"
```
Comment (fengmk2) [inline package.json:12]: It also work on 0.8+

**Comment by fengmk2** [comment]
@supershabam Please take a look on this when you are free.

**Comment by supershabam** [comment]
Thanks for the update. Looks good.

**Comment by fengmk2** [comment]
@supershabam Can you publish a new version to npm?

**Comment by supershabam** [comment]
is published

**Pull request merged**